openstack/identity/v3/projects/urls.go

```diff
@@ -0,0 +1,7 @@
+package projects
```

Comment:
This is how they do it and my plan is to push it back up so I need to follow their conventions.